WalletWasabi.Gui/Tabs/WalletManager/HardwareWallets/ConnectHardwareWalletViewModel.cs

```diff
@@ -390,14 +390,14 @@ public async Task LoadWalletAsync()
 					// Successfully initialized.
 					Owner.OnClose();
 				}
+				catch (OperationCanceledException ex)
+				{
+					Logger.LogTrace(ex);
+				}
 				catch (Exception ex)
 				{
-					// Initialization failed.
 					NotificationHelpers.Error(ex.ToUserFriendlyString());
-					if (!(ex is OperationCanceledException))
-					{
-						Logger.LogError(ex);
-					}
+					Logger.LogError(ex);
```

Comment:
Please log first and notify later, that's how it is everywhere else and it is imo the correct order.

Reply:
ACK

Reply:
ACK